typescript/sdk/src/metadata/ChainMetadataManager.ts

```diff
@@ -106,6 +106,15 @@ export class ChainMetadataManager<MetaExt = {}> {
     return chainMetadata;
   }
 
+  getMaxBlockRange(chainNameOrId: ChainNameOrId): number {
+    const metadata = this.getChainMetadata(chainNameOrId);
+    return Math.max(
+      ...metadata.rpcUrls.map(
+        ({ pagination }) => pagination?.maxBlockRange ?? 2000,
```

Comment:
Move this 2k to a const. Also IMO 2k is too big and many public providers won't support it. I'd go with 1k

Reply:
I thought 2k was the max for most